packages/pulumi/CHANGELOG.md

```
@@ -1,4 +1,12 @@
  
+## [10.0.1](https://github.com/tripss/nx-extend/compare/pulumi@10.0.0...pulumi@10.0.1) (2025-10-24)
+
+
+
+## [10.0.1](https://github.com/tripss/nx-extend/compare/pulumi@10.0.0...pulumi@10.0.1) (2025-10-24)
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)